<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonDialog</class>
 <widget class="QDialog" name="PersonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saisir les informations d'une personne</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>30</y>
     <width>95</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="firstnameFormLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="firstnameLabel">
      <property name="text">
       <string>Prénom :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="firstnameLineEdit">
      <property name="text">
       <string>Brice</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="photoLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel { background-color : black; color : red; }
</string>
   </property>
   <property name="text">
    <string>Aucune photo choisie.</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="photoPathLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>143</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cliquez pour choisir une photo ---&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>80</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="endFormLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="endLabel">
      <property name="text">
       <string>Fin d'intervention :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QDateEdit" name="endDateEdit">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="photoPathLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Url photo (120 x 120) :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="previewLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Aperçu photo :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="photoPathPushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choisir...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="companyFormLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="companyLineEdit">
      <property name="text">
       <string>Inconnue</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="companyLabel">
      <property name="text">
       <string>Societe :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>95</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="nameFormLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="text">
       <string>Jamin</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>30</y>
     <width>131</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="startFormLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="debutLabel">
      <property name="text">
       <string>Début d'intervention :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QDateEdit" name="startDateEdit">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PersonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PersonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
